content/en/blog/2025/mainframe-survey/index.md

```diff
@@ -0,0 +1,330 @@
+be available on the mainframe platform.
+
+![Prioritization of OpenTelemetry SDKs](q16.png)
+
+### Question 17: For which additional languages does your organization require an OpenTelemetry SDK?
+
+COBOL is the programming languages for the mainframe that most respondents (26)
+would like to have an OpenTelemetry SDK developed for. The SDK for COBOL is
+primarily requested by survey participants who have more than seven years of
+mainframe experience but also by five respondents which have less than three
+years of experience. More than 40 % of the respondents asked in the survey
+response for an SDK for REXX and JCL. Above one-fourth of the survey
+participants asked for an OpenTelemetry SDK for HLASM, and 20% for a PL/1 and C
+SDK. Three indicate an interest in an SDK for Metal C.
+
+![Demand for mainframe language support](q17.png)
+
+### Question 18: For which of the mainframe operating systems does your organization require the support of OpenTelemetry SDKs?
+
+In accordance with the operating systems utilized by the respondents, they
+express an interest in OpenTelemetry SDKs for these respective platforms. z/OS
+as supported platform for the OpenTelemetry SDKs is most important to the
+respondents (35), followed by Linux on IBM Z (13) and a single selection for
+zTPF.
+
+![Prioritization of Operating Systems to support OpenTelemetry SDKs](q18.png)
+
+### Question 19: What functionality of the OpenTelemetry Collector is most interesting for your organization to enable the processing and distribution of mainframe telemetry?
+
+The data collection functionality of the OpenTelemetry Collector is most
+important to the survey participants. Collection local to the source using the
+collector in an agent deployment (20) and from any system using receivers (19)
+scored highest in the responses. In addition, data aggregation for metrics is a
+high importance feature for the respondents (20). Data processing (15) and
+export (16), trace sampling (14) and the gateway deployment (14) are of interest
+for more than 30 % of the respondents. Hardware-based compression and encryption
+are important for nine of survey participants.
+
+![Prioritization of OpenTelemetry Collector functionality](q19.png)
+
+### Question 20: What use cases of system-level telemetry collection and processing do you envision for the OpenTelemetry Collector on the mainframe?
+
+When evaluating the OpenTelemetry Collector, the respondents prioritize the
+collection of system performance and platform metrics as the most important use
+case (30). The collection of system logs and mainframe support of resource
+detection are considered by around half of the survey participants as important.
+Data collection from Kubernetes and container runtimes is in focus by some of
+respondents and the use of the OpenTelemetry Collector for these use cases is of
+interest for them.
+
+![Prioritization of OpenTelemetry Collector telemetry collection by category](q20.png)
+
```

Comment:
Could you add a small conclusion here? Perhaps reiterating the invitation for others to become contributors?